The-SMIRNOFF-force-field-format.md

```diff
@@ -285,7 +285,7 @@ This decoupling of how parameters are assigned for each term provides a great de
 
 Below, we describe the specification for each force field term definition using the XML representation of a SMIRNOFF force field.
 
-As an example of a complete SMIRNOFF force field specification, see the prototype [SMIRNOFF99Frosst offxml](https://github.com/openforcefield/openforcefield/blob/master/openforcefield/data/test_forcefields/smirnoff99Frosst.offxml).
+As an example of a complete SMIRNOFF force field specification, see [a recent force field in the "Parsley" line (openff-1.2.0.offxml)](https://github.com/openforcefield/openforcefields/blob/1.2.0/openforcefields/offxml/openff-1.2.0.offxml).
```

Comment:
I modified `test_readme_links` to point to this file and found the following links to be broken

https://open-forcefield-toolkit.readthedocs.io/en/latest/converting.html
https://github.com/pandegroup/openmm/blob/topology/wrappers/python/simtk/unit/unit_definitions.py
https://mulan.swmed.edu/group/gaff.php
http://rdkit.org/ (HTTP?)

Might be nice to add that automated test, but not blocking.

Reply:
Thanks for checking. Replaced like 30 `http://`s with `https://`s. Found the new GAFF link. Pointed to more up-to-date/existant examples.